### ISSUE TITLE
Error: Encountered odd length

### ISSUE THREAD

**Issue by wkoszek** [suyashkumar/dicom#65]
Another issue when scanning many DICOMs:

```
Encountered odd length (vl=13945) when reading explicit VR SQ for tag (6003,1010)[private] (file offset 1752)
Encountered odd length (vl=27975) when reading explicit VR SQ for tag (6003,1010)[private] (file offset 1752)
Encountered odd length (vl=286383071) when reading implicit VR 'UN' for tag (1111,11e3)[private] (file offset 5382)
Encountered odd length (vl=419395) when reading explicit VR OW for tag (6000,3000)[??] (file offset 13496)
```

**Comment by suyashkumar** [comment]
Thanks for raising this! I will need to check further into the DICOM spec to see when we have odd value lengths, and why/when we expect them to be even (in the code, there is an explicit check to see if it's even).

**Comment by wkoszek** [comment]
@suyashkumar Just wanted to make an intro here to Brian Hardell here, aka @IChocked at Github.

Brian will intern with us this support, and one of his tasks is stress-testing our data ingest pipeline at Segmed. I've mentioned getting some data samples for all the recent reported issues is high on our list, so you might be seeing some DICOMs from us.

**Comment by IChocked** [comment]
I encountered this issue in some public data and wanted to share a few examples files.

[examples.zip](https://github.com/suyashkumar/dicom/files/4683909/examples.zip)

Data Citation:
_Kinahan, Paul; Muzi, Mark; Bialecki, Brian; Coombs, Laura. (2017). Data from ACRIN-FLT-Breast. The Cancer Imaging Archive. http://doi.org/10.7937/K9/TCIA.2017.ol20zmxg_

**Comment by suyashkumar** [comment]
These images all also seem to parse correctly with no errors with my quick fix in PR #79--there is a possible issue with the embedded icon image parsing though I want to look into (or maybe, a lot of the icons in some of these images are just empty, but will confirm after looking a bit deeper).
